### ISSUE TITLE
Cleanup/removal of BBB code in pi.microblog 

### ISSUE THREAD

**Issue by mgrbyte** [ploneintranet/ploneintranet#874]
@ale-rt Do you know if [this](https://github.com/ploneintranet/ploneintranet/blob/master/src/ploneintranet/microblog/statusupdate.py#L148-L164) BBB code can be removed now?

The proceeding method `getCharset` also has a BBB (in its docstring!), which to my reading at least is no longer applicable (?)

**Comment by ale-rt** [comment]
About the first part: I think it can be removed being that #506 was closed long time ago.
As far as I can remeber it was there in order to be compliant with an interface.

The getCharset method function should also be removed.
The BBB points to this code:
- https://github.com/ploneintranet/ploneintranet/blob/251c8cf9f1e69c38030b6b6ac2f7c93c86ae1e60/src/ploneintranet/microblog/browser/attachments.py#L45  # noqa
  Which is no longer there...

So my suggestion is to make a PR with those three functions removed and see:
- if you can post a status update with an image and download it
- if jenkins complains :)

**Comment by mgrbyte** [comment]
@ale-rt I'm sure jenkins will complain, the butler appears to be on strike/had his :heart broken ;)

I'm working on other things right now, but hopefully I can put in a PR over the next week or two.
Thanks for the quick feedback!

**Comment by ale-rt** [comment]
On Wed, Nov 11, 2015 at 3:48 PM, Matt Russell notifications@github.com
wrote:

> @ale-rt https://github.com/ale-rt I'm sure jenkins will complain, the
> butler appears to be on strike/had his :heart broken ;)
> 
> I'm working on other things right now, but hopefully I can put in a PR
> over the next week or two.
> 
> I will try to do the PR myself!

BTW, @gyst what is the status of jenkins? I have several PR waiting to be

## tested.

Alessandro Pisa - RedTurtle Technology
E-mail: alessandro.pisa@redturtle.it
Web Site: http://www.redturtle.it
Phone: +39 0532 1915958

**Comment by gyst** [comment]
Yes that crap can go but it only makes sense to do that when Jenkins is reliable. I don't know current status of that. Maybe better is @mattss integrates this cleanup in the content updates he's working on - which finalizes the work started with these deprecationwarnings.
